_data/ome2021.json

```diff
@@ -189,7 +189,8 @@
             "@type": "Organization",
             "name": "Micelio"
           },
-          "sameAs": "https://www.micelio.be/people.html"
+          "sameAs": "https://www.micelio.be/people.html",
+          "sameAs": "https://twitter.com/andrawaag"
```

Comment:
This is flagged as duplicate unique property and will need to be reformatted as

```
          "sameAs": [
            "https://www.micelio.be/people.html",
            "https://twitter.com/andrawaag"
          ]
```

Also https://github.com/ome/www.openmicroscopy.org/blob/8ac24418ef3c8f2eee7fcd139f82e9457b8017ed/_includes/ome2021_program_agenda.html#L24 will need to be updated to loop over all values and filter the Twitter URLs.